src/util.py

```diff
@@ -1,41 +1,33 @@
-"""
-util
-
-authors:
-@markoprodanovic
+"""Local utility module that handles printing error and success messages.
 
-last edit:
-Monday, January 12, 2020
+This module exports the following functions:
+    print_error - prints and formats an error message.
+    print_success - prints and formats a success message.
 """
+# authors:
+# @markoprodanovic
```

Comment:
you can add your name here too :)

Reply:
Name or github username? Also shouldn't your name be here too?

Reply:
up to you - I haven't added anything to the util file